backend/apps/cabins/validators.py

```diff
@@ -75,7 +75,7 @@ def checkin_validation(check_in, check_out, cabin_ids):
 
 
 def email_validation(email: str):
-    if not email.count("@") == 1 and "." not in email.split("@")[-1]:
```

Comment:
Previously, this would accept emails such as `example.example` as `not email.count("@") == 1` is `True` while `"." not in email.split("@")[-1]` would be `False` as `email.split("@")[-1]` is just `example.example`